include/crm/common/logging.h

```diff
@@ -66,7 +66,10 @@ crm_extended_logging(int t, int e)
 }
 #endif
 
+// @COMPAT Make internal when we can break API backward compatibility
+//! \deprecated Do not use
```

Comment:
I really hoped we'd get around to deprecating more of the logging code :/ Time marches on.